mars/dataframe/expressions/arithmetic/add.py

```diff
@@ -55,18 +58,37 @@ def fill_value(self):
     def scalar_value(self):
         return self._scalar_value
 
+    @property
+    def lhs(self):
+        return self._lhs
+
+    @property
+    def rhs(self):
+        return self._rhs
+
+    def _set_inputs(self, inputs):
+        super(DataFrameAdd, self)._set_inputs(inputs)
+        if len(self._inputs) == 2:
+            self._lhs = inputs[0]
```

Comment:
Let `self._lhs = self._inputs[0]` seem better, because in super's `_set_inputs`, the entity will be transformed into data.